core/templates/pages/story-editor-page/modal-templates/new-chapter-title-modal.controller.ts

```diff
@@ -108,6 +112,7 @@ angular.module('oppia').controller('CreateNewChapterModalController', [
       $scope.errorMsg = null;
       $scope.invalidExpId = false;
       $scope.correctnessFeedbackDisabled = false;
+      $scope.categoryNotDefault = false;
```

Comment:
Just so I understand better, what is a "default" category?

Reply:
A category is called 'default' if it is defined in the [constants.ts](https://github.com/oppia/oppia/blob/ff0f6f91d65025a1d5123690ba342e3173d97339/assets/constants.ts#L33-L34) file.

Reply:
Thanks! Maybe better to call this boolean `categoryIsDefault`?